source/production/F0.Minesweeper.Components/Layout/Footer.razor

```diff
@@ -1,7 +1,38 @@
 <footer id="f0-footer">
-    <hr />
-    <span>Follow this project on </span>
-    <a href="https://github.com/Flash0ver/F0.Minesweeper" target="_blank">
-        GitHub
-    </a>
+	<hr />
+	<span>Follow this project on </span>
+	<a href="https://github.com/Flash0ver/F0.Minesweeper" target="_blank">
+		GitHub
+	</a>
+
+	@if (ProductName is not null && ProductVersion is not null)
+	{
+		<p>@ProductName v@(ProductVersion)</p>
```

Comment:
`ProductVersion` = `$(VersionPrefix)-$(VersionSuffix)+$(RevisionId)`
let's use the commit-sha and link to the git-commit